packages/atlas/src/providers/auth/auth.provider.tsx

```diff
@@ -196,25 +214,43 @@ export const AuthProvider: FC<PropsWithChildren> = ({ children }) => {
           }
         }
 
-        SentryLogger.error('Unsupported error when posting login action', 'useLogIn', error)
+        SentryLogger.error('Unsupported error when posting login action', 'auth.provider', error)
         return {
           data: null,
           error: LogInErrors.LoginError,
         }
+      } finally {
+        setIsAuthenticating(false)
       }
     },
-    [saveEncodedSeed, setAnonymousUserId]
+    [refetch, saveEncodedSeed, setAnonymousUserId, setApiActiveAccount]
   )
 
+  const handleLogout = useCallback(async () => {
+    try {
+      await axios.post(`${ORION_AUTH_URL}/logout`, {}, { withCredentials: true })
+      handleAnonymousAuth(anonymousUserId).then((userId) => {
+        setAnonymousUserId(userId ?? null)
+      })
+      refetch()
+      setCurrentUser(null)
+      setEncodedSeed(null)
```

Comment:
should we await the refetch? Did you do this deliberately? I mean, did you want to set the current user and seed to null regardless?